src/frontend/src/lib.rs

```diff
@@ -94,3 +95,8 @@ pub async fn start(opts: FrontendOpts) {
     let session_mgr = Arc::new(SessionManagerImpl::new(&opts).await.unwrap());
     pg_serve(&opts.host, session_mgr).await.unwrap();
 }
+
+lazy_static::lazy_static! {
+    /// A compute client pool
+    pub static ref COMPUTE_CLIENT_POOL: ComputeClientPool = ComputeClientPool::new(1024);
```

Comment:
No lazy static 🤣. This will definitely cause problems for the playground mode, as all components are running in the same process. I'd prefer to pass it down, or having a pool for every component.

Reply:
Good catch! Since communication with CN all happens only when scheduling tasks, I'll pass it down for now.

Reply:
Maybe no need to pass it down. We can simply have a pool for batch scheduling, and another one for streaming exchanges, and yet another one for exchange.

Reply:
So now, for example, we create a pool in `FrontendEnv` for FrontEnd only and create another one in `LocalStreamManagerCore` for LocalStreamManager only, right?

The RPC service of scheduling and batch exchanges was wrapped inside `ComputeClient` together.